blobs.go

```diff
@@ -65,49 +61,78 @@ func (blobsTable) Schema() sql.Schema {
 	return BlobsSchema
 }
 
-func (r *blobsTable) TransformUp(f sql.TransformNodeFunc) (sql.Node, error) {
-	return f(r)
+func (r *blobsTable) WithFilters(filters []sql.Expression) sql.Table {
+	nt := *r
```

Comment:
I've seen this in the rest of the tables too, why can't `r` be used directly?

Reply:
Because it returns a new instance. By deref-ing the table, it makes a copy, so we don't have to manually create a table with all the fields, just modify the ones needed in the copy and return it as a pointer.
If in the future the table has more fields, we don't have to go to these methods and add the fields to the new instance.